src/python/pants/engine/process.py

```diff
@@ -489,6 +491,7 @@ def from_process(
             append_only_caches=process.append_only_caches,
             immutable_input_digests=process.immutable_input_digests,
             keep_sandboxes=keep_sandboxes,
+            working_directory=process.working_directory,
```

Comment:
This is a subtle breaking change for plugin authors if they're currently setting `working_directory` but its getting ignored. I guess I could mention that in the release notes?

Reply:
I think it's fine to treat this as a bugfix and mention it in the release notes.